<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatasetGeneratorUI</class>
 <widget class="QDialog" name="DatasetGeneratorUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>857</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dataset Generator</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>1020</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>610</y>
     <width>661</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Select SWC file:</string>
   </property>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>511</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>90</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>491</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Saving filename:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Use default saving filename</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>990</y>
     <width>351</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Parameter settings:</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Scaling factor on x: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>371</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Scaling factor on y: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Scaling factor on z: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Scaling factor for radius: </string>
    </property>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>961</width>
     <height>561</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Create Training Images</string>
    </attribute>
    <widget class="QTreeView" name="treeView">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>281</width>
       <height>481</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QToolBox" name="toolBox">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>20</y>
       <width>601</width>
       <height>481</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>601</width>
        <height>415</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Training Images Preparation</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>311</width>
         <height>80</height>
        </rect>
       </property>
       <property name="title">
        <string>3D Training Patches</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>300</y>
         <width>311</width>
         <height>80</height>
        </rect>
       </property>
       <property name="title">
        <string>2D Training Patches</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>581</width>
         <height>181</height>
        </rect>
       </property>
       <property name="title">
        <string>Image Source</string>
       </property>
       <widget class="QCheckBox" name="checkBox_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>201</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>From 3D image stack</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>161</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>From existing patches</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_4">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>30</y>
          <width>281</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>TeraFly format</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_7">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>130</y>
          <width>481</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
       </widget>
       <widget class="Line" name="line">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>100</y>
          <width>281</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>251</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Select folder or file from the left pannel</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>130</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>601</width>
        <height>415</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Neuron Structure File</string>
      </attribute>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Create List</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <signal>toggled(bool)</signal>
  <slot>filePath()</slot>
  <slot>okClicked()</slot>
  <slot>checkboxToggled(bool)</slot>
 </slots>
</ui>
